remote-state.tf

```diff
@@ -0,0 +1,20 @@
+resource "random_string" "resource_code" {
```

Comment:
We wouldn't usually track the storage account for the terraform state in terraform - you can't safely create the account until there's somewhere to store state, and obviously you can't create the state until the storage account exists. If you now tried to rename one of these resources, causing a destroy/recreate, then we'd lose the existing state. Curious to know if you disagree and there is some clever workaround/solution for this